pkg/gather/clusterconfig/ovs_flows.go

```diff
@@ -0,0 +1,62 @@
+package clusterconfig
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2"
+
+	_ "k8s.io/apimachinery/pkg/runtime/serializer/yaml"
+
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+const sdnNamespace = "openshift-sdn"
+const ovsCommand = "ovs-ofctl -O OpenFlow13 dump-flows br0"
+
+// GatherOVSFlows collects OVS flow information for every OVS pod in the openshift-sdn namespace
+//
+// Location in archive: networking/ovs_flows/
```

Comment:
Any thoughts about this location? I think it would be better to put it here and keep only pure Kubernetes/OpenShift stuff in the `config`.

Reply:
But maybe it's more confusing....not sure

Reply:
I think that makes sense to not store on the same path that we are storing k8s/os stuff. If we started to put some 'custom' gathers we may not want to mess up the data sources.